modules/sugar/src/test/ts/browser/ShadowDomTest.ts

```diff
@@ -8,32 +8,37 @@ import { PlatformDetection } from '@ephox/sand';
 import * as ShadowDom from 'ephox/sugar/api/node/ShadowDom';
 import { Testable } from '@ephox/dispute';
 import { htmlBlockTagName, htmlInlineTagName } from 'ephox/sugar/test/Arbitrary';
-import { withIframe, withNormalElement, withShadowElement } from 'ephox/sugar/test/WithHelpers';
+import {
+  withIframe,
+  withNormalElement,
+  withShadowElement,
+  withShadowElementInMode,
+  setupShadowRoot
+} from 'ephox/sugar/test/WithHelpers';
 import { tElement } from 'ephox/sugar/test/ElementInstances';
 import * as Document from 'ephox/sugar/api/node/Document';
 import * as Head from 'ephox/sugar/api/node/Head';
 import * as Body from 'ephox/sugar/api/node/Body';
 import * as Node from 'ephox/sugar/api/node/Node';
+import * as DomEvent from 'ephox/sugar/api/events/DomEvent';
+import { Remove } from '@ephox/sugar';
```

Comment:
This import should be `import * as Remove from 'ephox/sugar/api/dom/Remove';`

Reply:
Fixed.